cypress/integration/dictionary/addConceptsToDictionary.feature

```diff
@@ -14,7 +14,8 @@ Feature: Add concepts to an existing dictionary
   @dictionary
   @ciel
   Scenario: The user should be able to add a single concept from their preferred source
-    Given the user is on the "Import existing concept" page
+    Given a dictionary exists
```

Comment:
is there any good way how we can refactor these scenarios, Especially scenarios which look the same.

Reply:
Yeah I think this "Given a dictionary exist" step belongs in the backgroun of these tests

Reply:
I actually think this is fine. In general, I'm happy with the feature files being a little repetitive, as each feature should uniquely describe the steps it takes to use that feature, even if they are obvious or done everywhere.